### ISSUE TITLE
add framework

### ISSUE THREAD

**Issue by incanus** [mapbox/DEPRECATED-mapbox-ios-sdk#510]
Add iOS 8+ dynamic framework, which among other things will allow Swift integration.

**Comment by bilby91** [comment]
@incanus Mapbox should work with Swift through bridging headers right ?

**Comment by incanus** [comment]
Sorry, yes, I was confusing straight Swift use with Swift Playgrounds, which require frameworks. Still would like a framework though for things like Extension sharing and just easier setup. Compare steps in [`mapbox-gl-cocoa/dist/README.md`](https://github.com/mapbox/mapbox-gl-cocoa/blob/master/dist/README.md), for example.

**Comment by incanus** [comment]
https://github.com/incanus/Swiftbox is a bridging header example for Swift.

**Comment by Cleitaro** [comment]
@incanus Do you plan to release a new version with dynamic framework for easier setup soon?

**Comment by incanus** [comment]
@Cleitaro Yes, but it hasn't been a priority lately. I'll try to hit this soon.

**Comment by Cleitaro** [comment]
@incanus Thanks, looking forward to it!

**Comment by zilongqiu** [comment]
@incanus Thanks man !
